linear-regression.ipynb

```diff
@@ -105,7 +105,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Can you comment on anything?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rowanc1/ubc-demo/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>